packages/block-library/src/navigation/style.scss

```diff
@@ -171,7 +171,6 @@ $navigation-sub-menu-width: $grid-unit-10 * 25;
 		}
 
 		&.has-child .wp-block-navigation-link__content {
-			min-width: 100%;
```

Comment:
This fixes an issue with the submenu icon overlaying the next menu item: 

<img width="541" alt="Screen Shot 2020-04-06 at 2 59 30 pm" src="https://user-images.githubusercontent.com/8096000/78528396-9d35ee80-7822-11ea-857b-fde18c967fae.png">